src/vs/platform/encryption/electron-main/encryptionMainService.ts

```diff
@@ -0,0 +1,60 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { safeStorage } from 'electron';
+import { IEncryptionMainService } from 'vs/platform/encryption/common/encryptionService';
+import { ILogService } from 'vs/platform/log/common/log';
+
+export class EncryptionMainService implements IEncryptionMainService {
+	_serviceBrand: undefined;
+
+	constructor(
+		private readonly machineId: string,
+		@ILogService private readonly logService: ILogService
+	) { }
+
+	async encrypt(value: string): Promise<string> {
+		return JSON.stringify(safeStorage.encryptString(value));
```

Comment:
Curious, what is the purpose of converting the encrypted data to JSON string in the new API ?

Reply:
Oh I just wanted to have it return a string and went with JSON.stringify as the way to do that.

Reply:
Given you are using Buffer api to convert back in decrypt, why not `Buffer::toString` ?